server.js

```diff
@@ -99,7 +99,9 @@ app.get( '/avatar/:emailHash', async ( request, response ) => {
         return false;
     }
 
-    if ( request.params.emailHash.length !== 32 && request.params.emailHash.length !== 64 ) {
+    const emailHash = request.params.emailHash.toLocaleLowerCase();
```

Comment:
Is this spec compliant?

Reply:
where did you find that?

Reply:
forgot to push :(

Reply:
No, I'm just curious what the spec says about lowercase & uppercase

Reply:
Dosen't say anything in the spec that the server should lowercase it. But the API specify that the client should lowercase it.

Reply:
Id suggest not lowercasing then in case we want to handle some sort of special casing down the line

Reply:
yes, we are commenting on an outdated diff